headers/types/common/common.h

```diff
@@ -36,6 +36,26 @@ struct file_stream {
 };
 ASSERT_SIZE(struct file_stream, 72);
 
+// These flags are shared with the function to display text inside message boxes
+// So they might need a rename once more information is found
+struct preprocessor_flags {
+    uint16_t unknown0 : 13;
+    bool show_speaker : 1;
+    uint32_t unknown18 : 18;
+};
+ASSERT_SIZE(struct preprocessor_flags, 4);
+
+// Represents arguments that might be passed to the PreprocessString function
+struct preprocessor_args {
+    uint32_t flag_vals[4]; // These are usually IDs with additional flags attached to them
+    uint32_t id_vals[5];
+    int32_t number_vals[5]; 
```

Comment:
[Formatter caught]([Formatter caught](https://github.com/UsernameFodder/pmdsky-debug/runs/5298100158?check_suite_focus=true) that there's enough room to put `struct preprocessor_flags flags` on the previous line.) that there's an extra space at the end of this line :o.